packages/mwp-tracking-plugin/src/click.test.js

```diff
@@ -40,7 +40,12 @@ describe('onPreHandlerExtension', () => {
 		const injectRequest = {
 			method: 'get',
 			url: '/foo',
-			credentials: 'whatever',
+			auth: {
+				credentials: {
+					user: 'whatever',
+				},
+				strategy: 'bypass',
```

Comment:
> In order to fully support request injection authentication as a production-ready feature, the injection options were changed to require passing the name of the strategy being simulated to authenticate the request. Previously, a special 'bypass' strategy name was set for all injected credentials. To use the new format simply wrap the credentials and optional artifacts with an auth object and add a new strategy key with a name matching a configured authentication strategy.

Basically, we still want to bypass auth, so updated the credentials object for every `server.inject()` to explicitly pass the `bypass` strategy. Not recommended for prod code, but fine for testing.